client/src/main/java/org/apache/rocketmq/client/consumer/DefaultLitePullConsumer.java

```diff
@@ -315,7 +315,9 @@ public long getAutoCommitIntervalMillis() {
     }
 
     public void setAutoCommitIntervalMillis(long autoCommitIntervalMillis) {
-        this.autoCommitIntervalMillis = autoCommitIntervalMillis;
+        if (autoCommitIntervalMillis >= 1000) {
```

Comment:
It would be better if you can replace this magic number(1000) with a constant variable.